tests/test_dynamodb_log_based.py

```diff
@@ -27,6 +27,8 @@ def generate_items(self, num_items, start_key=0):
             record = {
                 'int_id': i,
                 'string_field': self.random_string_generator(),
+                # BUG https://jira.talendforge.org/browse/TDL-19094
```

Comment:
Has this issue been pointed?

Reply:
No, do I need to give it a review label?

Reply:
yep!

Reply:
Done!